templates/agent-conf.d/disk.yaml.erb

```diff
@@ -0,0 +1,28 @@
+init_config:
```

Comment:
Thanks, Bruce. These are actually direct copies of the ones in the DD repo. I would rather leave them as is. Would actually rather use that repo instead of ours, if at all possible (separate ticket).